### PR TITLE
add multilabel classification

### PR DESCRIPTION
**Pull request by sarakolding**
# Notes for reviewers
Attempted to allow multilabel classification without breaking current code current code :o) 
I should probably make some tests?

### CODE REVIEW

**Comment by sarakolding** [comment]
får static type check fails i jakob og eriks project... skal jeg ændre det for dem?

File: psycop/common/model_training/training/train_and_predict.py [hunk 1]
```diff
@@ -88,6 +93,74 @@ def stratified_cross_validation(
...
+    # Create folds
+    msg.info("Creating folds")
+    msg.info(f"Training on {X.shape[1]} columns and {X.shape[0]} rows")
+
+    # stratify by first outcome column
+    folds = StratifiedGroupKFold(n_splits=cfg.n_crossval_splits).split(
+        X=X,
+        y=y[outcome_col_name[0]],
+        groups=train_df[cfg.data.col_name.id],
+    )
```
Comment (HLasse) [inline psycop/common/model_training/training/train_and_predict.py:123]: to stratify across all outcomes: concanate the output columns into a single column (an example row would be e.g. "0001" - stratify on this column

File: psycop/common/model_training/utils/col_name_inference.py [hunk 2]
```diff
@@ -30,20 +30,24 @@ def get_col_names(
...
+        outcome_col_name = potential_outcome_col_names
 
     train_col_names = [
         c
         for c in dataset.columns
         if c.startswith(cfg.data.pred_prefix) and "uuid" not in c
     ]
 
-    return outcome_col_name, train_col_names
+    return outcome_col_name, train_col_names  # type: ignore
```
Comment (HLasse) [inline psycop/common/model_training/utils/col_name_inference.py:50]: hvorfor brokker den sig over types her?

Comment (sarakolding) [reply psycop/common/model_training/utils/col_name_inference.py]: det gør den vist heller ikke, så det var bare en fejl

File: psycop/common/model_training/training/train_and_predict.py [hunk 3]
```diff
@@ -88,6 +93,76 @@ def stratified_cross_validation(
...
+    msg = Printer(timestamp=True)
+
+    X = train_df[train_col_names]
+    y = train_df[outcome_col_name]
+
+    # Create folds
+    msg.info("Creating folds")
+    msg.info(f"Training on {X.shape[1]} columns and {X.shape[0]} rows")
+
+    y_joined = y.astype(int).astype(str).apply("".join, axis=1)
```
Comment (HLasse) [inline psycop/common/model_training/training/train_and_predict.py:118]: make a comment as to why this line is here

**Review by HLasse — APPROVED** [review]

**Pull request merged**